lib/demangler-win32.js

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2017, Patrick Quist
```

Comment:
I didn't write this ^^

Reply:
haha, copied the original file from cl-support, and then completely rewrote it :P